src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { useAppDispatch, useAppSelector } from '../../hooks';
+import { setCurrentTodo } from '../../features/currentTodo';
+import classNames from 'classnames';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoItem: React.FC<Props> = ({ todo }) => {
+  const dispatch = useAppDispatch();
+  const currentTodo = useAppSelector(state => state.currentTodo);
+  const { id, title, completed } = todo;
+
+  return (
+    <tr data-cy="todo">
+      <td className="is-vcentered">{id}</td>
+      <td className="is-vcentered">
+        {completed && (
+          <span className="icon" data-cy="iconCompleted">
+            <i className="fas fa-check" />
+          </span>
+        )}
+      </td>
+
+      <td className="is-vcentered is-expanded">
+        <p className={completed ? 'has-text-success' : 'has-text-danger'}>
```

Comment:
Use the classnames library rules for add classes with condition, fix it everywhere